deploy/kubernetes/charts/monero/templates/deployment.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "monero.fullname" . }}
+  labels:
+    {{- include "monero.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "monero.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "monero.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
```

Comment:
This is a declared value, yet its empty. Is that convention, future-proofing, or does this error without *a* definition being provided?

Reply:
it is for future-proofing. It will be used in the future. I can delete it for now, if it would be better.

Reply:
:+1: